app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+import shutil
+import os
+
+
+def move_file(command: str) -> None:
+    commands = command.split()
+
+    if len(commands) == 3 and command.startswith("mv "):
+        _, first_file_name, path_to_second_file = commands
+        (path_to_second_file,
+            second_file_name) = os.path.split(path_to_second_file)
+
+        if path_to_second_file == "":
```

Comment:
Using `if not path_to_second_file` will make the condition more readable and general

Reply:
I don't know why, but yesterday it didn't work. Today is OK.